modules/kafka/kafka-dest-worker.c

```diff
@@ -340,6 +351,26 @@ _thread_init(LogThreadedDestWorker *s)
   return log_threaded_dest_worker_init_method(s);
 }
 
+static gboolean
+kafka_dest_worker_connect(LogThreadedDestWorker *s)
+{
+  KafkaDestWorker *self = (KafkaDestWorker *)s;
+  KafkaDestDriver *owner = (KafkaDestDriver *) self->super.owner;
+
+  static gboolean first_init = FALSE;
+  if (!first_init)
+    {
+      first_init = TRUE;
```

Comment:
This branch expects that kafka_dd_init already initialized the librdkafka client.
Can we do an assert here at first_init or an explicit check on owner->kafka object ?